addons/bestja_project/views/project.xml

```diff
@@ -7,8 +7,10 @@
             <field name="view_mode">kanban,tree,form</field>
             <field name="domain">[
                 '|',
-                    ('organization.coordinator.id', '=', uid),
-                    ('manager.id', '=', uid),
+                    ('organization.coordinator', '=', uid),
+                '|',
```

Comment:
Yes, it does. I think the diff here may be hard to read. This is the domain before:
```
[
    '|',
        ('organization.coordinator.id', '=', uid),
        ('manager.id', '=', uid),
]
```

And this is the domain after:
```
[
    '|',
        ('organization.coordinator', '=', uid),
    '|',
        ('manager', '=', uid),
        ('members', '=', uid),
]
```
Basically this change adds a new rule - `('members', '=', uid),`. Note that Odoo domains use Polish notation - that's why the new `|` operator went where it went.
The removal of `.id` is just a style change - I change my style preference in the meantime.